test/smoke/sentinel.js

```diff
@@ -21,21 +28,54 @@ describe('Sentinel', () => {
           }),
         };
 
-        const containers = await docker.listContainers(options);
+        const requestListContainers = docker.listContainers(options)
+          .then((result) => {
+            listContainers[hostName] = result;
```

Comment:
You can do everything in one step and get container log right here

Reply:
Can you explain a bit more what you mean? The promises can't be resolved at the same time because the second promise depends on data from the first promise existing. I tried adding the call to get the lists in another `.then()` statement, but I can't figure out how to do this so it all resolves as a single promise...

Reply:
@shumkov I think I got it working in a single promise now?